public/css/main.css

```diff
@@ -119,10 +123,9 @@ span.auth-option {
   }
 
   .foot {
-    height: 180px;
+    height: 180px !important;
```

Comment:
The use of `!important` as a property value is strongly discouraged in the community. Is there no other way we can achieve this without using `!important`. We might want to increase the specificity of that element and then target it accordingly.